extensions/eventtap/init.lua

```diff
@@ -177,29 +177,41 @@ function module.rightClick(point, delay)
     module.event.newMouseEvent(module.event.types["rightMouseUp"], point):post()
 end
 
---- hs.eventtap.middleClick(point[, delay])
```

Comment:
What about keeping the middleClick wrapper (now just calling the other click) for backwards compatibility and as a usability matter as middleClick is a well known event (unambiguous name for clicking with mouse button 2).